src/button.mobile.js

```diff
@@ -28,8 +27,18 @@ var UIButtonMobile = function(log, settings, uiValidationUtils, $, gmApi, uiUtil
         }
         log.debug("Requirements checked, all ok");
         button = $(resources.getResource('button.html'));
-        gmApi.GM_addStyle(resources.getResource('button.css'));
-        gmApi.GM_addStyle(resources.getResource('selector.css'));
+
+        var css = document.createElement('style');
```

Comment:
That's weird, why removing the `gmApi`? You could've simply supplied your own implementation in the mobile build.